workers/unity/Assets/Gdk/Core/MutableView/MutableView.cs

```diff
@@ -14,13 +14,10 @@ public class MutableView
     {
         public const long WorkerEntityId = -1337;
 
-        public Dictionary<int, ComponentTranslation> TranslationUnits;
-        public Action<Entity, long> AddAllCommandRequestSenders;
+        public readonly Dictionary<int, ComponentTranslation> TranslationUnits = new Dictionary<int, ComponentTranslation>();
+        private Action<Entity, long> AddAllCommandRequestSenders;
```

Comment:
camelCase for the private variables and we can remove the `{get; }` part from the EntityManager and make it readonly instead

Reply:
Fixed 👍